java101/src/com/java101/PostacTest.java

```diff
@@ -0,0 +1,44 @@
+package com.java101;
```

Comment:
Przenieś to z /src/com/java101/PostacTest.java do /test/com/java101/PostacTest.java